NitroxClient/GameLogic/NitroxConsole.cs

```diff
@@ -48,32 +45,39 @@ public void Spawn(GameObject gameObject)
         }
 
         /// <summary>
-        /// Spawns a Seamoth or an Exosuit
+        /// Spawns Seamoth, Exosuit and Cyclops
         /// </summary>
-        private void SpawnVehicle(GameObject gameObject)
+        private void SpawnVehicle(GameObject gameObject, TechType techType)
         {
-            TechType techType = CraftData.GetTechType(gameObject);
-
             NitroxId id = NitroxEntity.GetIdOrGenerateNew(gameObject);
 
-            VehicleWorldEntity vehicleEntity = new VehicleWorldEntity(null, DayNightCycle.main.timePassedAsFloat, gameObject.transform.ToLocalDto(), "", false, id, techType.ToDto(), null);
-            VehicleChildEntityHelper.PopulateChildren(id, gameObject.GetFullHierarchyPath(), vehicleEntity.ChildEntities, gameObject);
-
+            VehicleWorldEntity vehicleEntity = Vehicles.BuildVehicleWorldEntity(gameObject, id, techType);
+            
             packetSender.Send(new EntitySpawnedByClient(vehicleEntity));
 
-            Log.Debug($"Spawning vehicle {techType} with id {techType} at {gameObject.transform.position}");
+            Log.Debug($"Spawning vehicle {techType} with id {id} at {gameObject.transform.position}");
         }
 
-        /// <summary>
-        /// Spawns a Pickupable item
-        /// </summary>
-        private void SpawnItem(GameObject gameObject)
+        private void DefaultSpawn(GameObject gameObject)
+        {
+            item.Dropped(gameObject);
```

Comment:
Can we rename `item` to `items`? It's not really part of this pr but would be better for readability nonetheless.